docs/docs/teams/list-teams.md

```diff
@@ -1,11 +1,12 @@
 ---
+sidebar_position: 2
 title: List Teams
 intro: View your list of teams
 links:
   overview:
   quickstart:
   previous: teams/add-team
-  next: teams/view-team
+  next: teams/members/index
```

Comment:
In the list of teams the next natural step should be click on a team to view its details. Don't you think?

```suggestion
  next: teams/view-team
  ```